### ISSUE TITLE
Need docs for pause and resume error

### ISSUE THREAD

**Issue by goriunov** [tulios/kafkajs#584]
Just found out that pause may throw error by reading tests and having multiple times crashes during my test runs. It would be good to have this  written somewhere:

```
   it('throws an error if Consumer#run has not been called', () => {
      expect(() => consumer.pause([{ topic: 'foo', partitions: [0] }])).toThrow(
        KafkaJSNonRetriableError,
        'Consumer group was not initialized, consumer#run must be called first'
      )
    })
```

Also as a side note IMHO would be better to return Promise instead of void so you can easily catch error without using try catch :)

**Comment by goriunov** [comment]
Looks like `resume` has the same behaviour

**Comment by JaapRood** [comment]
The error you're referring to is thrown when the consumer isn't running yet. In other words, when calling `consumer.resume` or `consumer.pause` before `consumer.run` has been, or after `consumer.stop`. And you're totally right, there is no explicit mention of this in the docs at the time of writing. 

I'll leave this enhancement of documentation open for a first collaborator 🙂.

**Comment by tulios** [comment]
The documentation is missing, but the behavior is correct. You shouldn't try to recover from these errors, as you can see from the message (`Consumer group was not initialized, consumer#run must be called first`). Currently, you can't call these methods before calling `consumer#run`.

**Comment by Nevon** [comment]
Fixed by #662
